### DISCUSSION TITLE
Escaping full stop/decimal characters in custom tailwind config

### DISCUSSION THREAD

**Discussion by andrew-ireland** [tailwindlabs/tailwindcss#7900] [Help] [answered]
When extending a standard tailwind config with additional custom spacing spacing classes, sometimes it's handy to add full stop/decimal values (similar to the default margin + padding scale. e.g. py-1.5). I've tried doing this with a custom 'body' spacing property, but have run into an error using names with full stop. Anyone know if there is an easy/recommended way to escape the full stop character?

I.e:
'body-1'      ✅
'body-1.5'   ❌
'body-2'     ✅

**Comment by adamwathan** [comment, marked as answer]
Hey! Should just work: https://play.tailwindcss.com/FRbvo2QOfO?file=config

**Comment by andrew-ireland** [comment]
🤔Hmmm, thanks for confirming — I thought it should (and had worked previously). Must be something going awry elsewhere...

**Comment by andrew-ireland** [comment]
Posting my answer, in case someone else stumbles across it: 

TLDR; class was getting stripped/purged in my template.

when I used `{{ unless last }}u-mb-body-1.5{{ /unless }}`, the class wasn't being generated.
when I used `{{ unless last }} u-mb-body-1.5 {{ /unless }}` the class is generated and works as expected.

Interestingly, once I've done the second, I can go back to the first.... but it has to be parsed with the space around it!
